### PR TITLE
Add a note about viewOnly.

### PR DESCRIPTION
**Pull request by dheap**

### CODE REVIEW

File: releasenotes/jmri4.99.4.shtml [hunk 1]
```diff
@@ -260,6 +260,11 @@ The <a href="https://github.com/JMRI/JMRI/issues?utf8=✓&q=milestone%3A4.99.4+i
         <a id="DecoderPro" name="DecoderPro"></a>
         <ul>
             <li>You can now write out just edited CV values to a CSV file.</li>
+            <li>The programmer pane "display" element now supports a "viewOnly" attribute.
+                For more information see the
+                <a href="/help/en/html/apps/DecoderPro/programmer.shtml" target="_blank">Create a Custom Programmer</a>
```
Comment (danielb987) [inline releasenotes/jmri4.99.4.shtml:265]: This URL https://www.jmri.org/help/en/html/apps/DecoderPro/programmer.shtml gives me a 404.

But this URL works:
https://www.jmri.org/help/en/html/apps/DecoderPro/Programmer.shtml

The error seems to be that `programmer.shtml` should be `Programmer.shtml`.

**Review by danielb987 — CHANGES_REQUESTED** [review]

**Comment by dheap** [comment]
Whoops. My local Mac Apache didn't balk at it.

**Comment by dheap** [comment]
@danielb987 Is this good to go?

**Review by danielb987 — APPROVED** [review]
Thanks!

**Pull request merged**